### PR TITLE
kona: new recipe

### PR DESCRIPTION
**Pull request by Crestwave**
I'll open a pull request to upstream for the patch after this gets merged.

### CODE REVIEW

File: dev-lang/kona/patches/kona-20190226.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+--- a/Makefile
++++ b/Makefile
+@@ -86,6 +86,14 @@ LDFLAGS += -lsocket
+ PRODFLAGS += -fast
+ endif
+ 
++ifeq (haiku,$(OS))
++CFLAGS += -lpthread
```
Comment (korli) [inline dev-lang/kona/patches/kona-20190226.patch]: Is this needed?

Comment (Crestwave) [reply dev-lang/kona/patches/kona-20190226.patch]: I'm not sure; I added it because the most of the targets had `-pthread`.

Comment (alaviss) [reply dev-lang/kona/patches/kona-20190226.patch]: Haiku libpthread is empty, and is only there for compatibility purposes I believe. You can safely remove it.

File: dev-lang/kona/kona-20190226.recipe [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+SOURCE_URI="https://github.com/kevinlawler/kona/archive/Win64-$portVersion.tar.gz"
+CHECKSUM_SHA256="26cfebd47a3ae60753fc8a910132f45cad58381155255fbac2129e4507fef403"
+SOURCE_DIR="kona-Win64-$portVersion"
+SOURCE_FILENAME="kona-$portVersion.tar.gz"
+PATCHES="kona-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
```
Comment (alaviss) [inline dev-lang/kona/kona-20190226.recipe]: You forgot to set this for non `x86_gcc2`

File: dev-lang/kona/patches/kona-20190226.patch (hunk 1 above)
Comment (Crestwave) [reply dev-lang/kona/patches/kona-20190226.patch]: Ah, ok. I've removed it now.

**Comment by alaviss** [comment]
You're force pushing a lot without any changes. Did you forgot to `git add` and/or `git commit`?

**Comment by Crestwave** [comment]
Sorry; my internet was down so I had to transfer it to my phone to push it and I think I confused some stuff up. It should be fixed now.

**Pull request merged**